packages/apps/docs/src/docs/build/nft-marmalade/storage.md

```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
 
 When you call the `get-uri` function and pass in a token ID, the function accesses the `tokens` map, finds the row corresponding to the provided token ID, and returns the value stored in the `'uri` field of that row. 
 
-## Leveraging Arweave with Akord
+## Using Arweave with Akord
 
-Given the immutable characteristic of NFTs, the associated data's long-term
-accessibility and sustainability are critical aspects to consider. 
-Akord has
-emerged as a powerful tool to handle these requirements, and its importance is
-underscored in the context of Marmalade.
+**Arweave** is a data storage blockchain, designed specifically to permanently store data on-chain with one upfront payment. As an on-chain storage solution, data is immutable and timestamped. Its permanence is guaranteed by an on-chain endowment, which ensures miners are incentivized to store data for a minimum of 200 years.
 
-Arweave is a pioneering protocol designed to modernize the internet's
-foundational structures, augmenting its capacity for storing, addressing, and
-accessing data. 
-This technology is central to our approach to dealing with NFT
-data, ensuring its perpetual availability and integrity and thereby
-strengthening the overall dependability of the NFTs hosted on Marmalade.
+**Akord** provides a web app, API, CLI and SDK to easily upload and manage data on Arweave. To use Arweave for storing your NFT assets, take the following steps.
 
-### What is Akord?
+### 1. Upload the image asset to Arweave
```

Comment:
```suggestion
### Upload assets with the Akord app
```

I might suggest getting rid of empty headings and bump up the content a heading level and use numbered steps for steps.